utils/insert_data_models_db_utils.py

```diff
@@ -0,0 +1,57 @@
+import os
+from hafifa.data_base import data_models
+from hafifa.data_base.SQLAlchemy import SQLAlchemyHandler
+from hafifa.utils import frame_utils, metadata_utils
+
+
+def create_and_insert_to_database_video_metadata_frame_models(path: str,
+                                                              image_list: list,
+                                                              video_file_name: str):
+    """
+    Create and insert to database video metadata and frame models.
+    :param path: Video path.
+    :param image_list: Image list.
+    :param video_file_name: Video file name.
+    """
+    db = SQLAlchemyHandler()
+    observation = get_observation_name_from_path(path)
+    video_from_db = db.get_video(observation, video_file_name, len(image_list))
+
+    if not video_from_db:
+        db.insert_one(data_models.Video(observation_name=observation, os_path=video_file_name, number_of_frames=len(image_list)))
+
+        video_model = db.get_video(observation_name=observation,
+                                   os_path=video_file_name,
+                                   number_of_frames=len(image_list))
```

Comment:
You dont have to make 2 requests to the db/
After inserting the model it should update its id to the one in the database.